addons/website/views/snippets/s_banner_connected.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_banner_connected" name="Banner connected">
+    <section
+        class="s_banner_connected o_cc o_cc5 oe_img_bg pt128 pb136 parallax"
+        data-oe-shape-data="{'shape':'web_editor/Connections/13','colors':{'c5':'#FFFFFF'},'showOnMobile':true}"
+        data-scroll-background-ratio="0.75"
+        data-parallax-type="zoom_in"
+        >
+        <span
+            class="s_parallax_bg oe_img_bg"
+            style="background-image: url('/web/image/website.s_banner_connected_default_image'); background-position: 50% 75%;"
+            />
+        <div
+            class="o_we_bg_filter"
+            style="background-image: radial-gradient(circle farthest-side at 25% 25%, rgba(222, 222, 222, 0) 0%, rgb(69, 69, 69) 100%);"
+            />
+        <div
+            class="o_we_shape o_web_editor_Connections_13 o_shape_show_mobile"
+            style="background-image: url('/web_editor/shape/web_editor%2FConnections%2F13.svg?c5=rgb%28255%2C255%2C255%29');"
```

Comment:
The color value on `data-oe-shape-data` and the shape URL should be the same (`rgb(255, 255, 255)`)... Also, I don't think the `/` needs to be encoded as `%2F`, browsers can handle it just fine in `url(...)` context :+1: